code/modules/vending/_vending.dm

```diff
@@ -690,7 +691,7 @@ GLOBAL_LIST_EMPTY(vending_products)
 			price = R.custom_price || default_price,
 			max_amount = R.max_amount,
 			ref = REF(R),
-			extended = TRUE
+			premium = TRUE
```

Comment:
This change needs explaining to me.

Reply:
This var wasn't used in the interface, and contraband items are also not exempt from department members as @actioninja mentioned, so their price needs to be shown too.

Reply:
yeah, this is correct.